src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -100,6 +100,10 @@ public boolean elevatorIsStowed() {
     return zeroed && encoder.getPosition() <= .5;
   }
 
+  public boolean elevIsAtPos() {
+    return (Math.abs((profiledPid.getGoal().position) - (encoder.getPosition())) >= 2);
```

Comment:
2 inches is a lot of error, and the profiled pid controller has an `atGoal` method that probably should be used instead. We'll probably need to set a tolerance on the PID controller using https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/controller/ProfiledPIDController.html#setTolerance(double). Looking at the graph from one of our match logs, the goal position of the elevator was 16.3 inches, and the real position was 16.36, so a tolerance of .1 should be fine. 
![Uploading Screenshot 2024-04-14 at 20.16.38.png…]()